src/LetsEncrypt/Internal/CertificateSelector.cs

```diff
@@ -89,7 +89,7 @@ private static void AddWithDomainName(ConcurrentDictionary<string, X509Certifica
 
             if (domainName == null || !_certs.TryGetValue(domainName, out var retVal))
             {
-                return _options.Value.FallbackCertificate;
+                return PreloadIntermediateCertificates(_options.Value.FallbackCertificate);
```

Comment:
This is going to run on every new SSL connection. Is there a way to do this once per certificate instead of once per SSL connection?

Reply:
oops, what a brainfart, will fix shortly